src/modules/PowerOCR/PowerOCR/App.xaml

```diff
@@ -3,8 +3,18 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:local="clr-namespace:PowerOCR"
-    ShutdownMode="OnExplicitShutdown"
+    xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml"
```

Comment:
This namespace is causing an issue with the CI Build. Giving an unknown namespace error. Likely related to the package downgrade?